pcdsdevices/areadetector/detectors.py

```diff
@@ -161,3 +163,51 @@ class PCDSAreaDetector(PCDSAreaDetectorEmbedded):
     stats5 = Cpt(StatsPlugin, 'Stats5:')
     tiff1 = Cpt(TIFFPlugin, 'TIFF1:')
     trans1 = Cpt(TransformPlugin, 'Trans1:')
+
+class PCDSAreaDetectorTyphos(Device):
+    """
+    A 'bare' PCDS areadetector class specifically for Typhos screens.
+    Implements only the most commonly used PVs for areadetector IOCS.
+    Includes a simple image viewer. 
+    """
+
+    # Status and specifications
+    manufacturer = Cpt(EpicsSignalRO, 'Manufacturer_RBV', kind='config')
+    camera_model = Cpt(EpicsSignalRO, 'Model_RBV', kind='normal')
+    sensor_size_x = Cpt(EpicsSignalRO, 'MaxSizeX_RBV', kind='config')
+    sensor_size_y = Cpt(EpicsSignalRO, 'MaxSizeY_RBV', kind='config')
+    data_type = Cpt(EpicsSignal, 'DataType', kind='config')
+    data_type_RBV = Cpt(EpicsSignalRO, 'DataType_RBV', kind='config')
+
+    # Acquisition settings
+    exposure = Cpt(EpicsSignal, 'AcquireTime', kind='config')
+    exposure_rbv = Cpt(EpicsSignalRO, 'AcquireTime_RBV', kind='config')
```

Comment:
Any reason to not combine the setpoint and readback (i.e., using `EpicsSignalWithRBV`)?

Reply:
@klauer Is "I didn't notice that signal type before" a good reason? :)

Let me try that out; that could clean this device (and the Qmini, for that matter) up quite a bit.

Reply:
A very good reason indeed! 😄

Reply:
Swapped to EpicsSignalWithRBV; this looks much better now. Thanks!